db/migrate/20200730023943_clean_up_people1.rb

```diff
@@ -0,0 +1,7 @@
+class CleanUpPeople1 < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :people, name: 'index_people_on_mal_id'
+    remove_index :people, name: 'person_mal_id'
+    remove_column :people, :mal_id
```

Comment:
Have these been loaded into the Mapping table?

Reply:
I am not sure how I could check that, wouldn't we have to run a query against prod to see if the `mal_id` exists under `myanimelist/people`?

Reply:
I just verified it ✅